src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/_command_summary.html.twig

```diff
@@ -20,8 +13,17 @@
         </h2>
 
         <dl class="metadata">
-            {% if status_code == 0 or interrupted %}
-                <span class="status-response-status-code">{{ interrupted is null ? 'Success' : 'Interrupted by signal: ' ~ interrupted }}</span>
+            {% if status_code == 0 %}
+                {% if interrupted %}
```

Comment:
Command can be interrupted with an exit code != 0.

Reply:
I've updated the logic. I hope I got it right this time. Thanks.

Reply:
Yes, let's do it. This is how it looks:

![interrupted-exit-code](https://github.com/symfony/symfony/assets/73419/3822ad58-3b3b-4746-b9f9-b7323a62eb1c)